### PR TITLE
Fix a bug with ignored error key

### LINKED ISSUE #209: Backend should not ignore error_key for single rule endpoint

### PR DESCRIPTION
**Pull request by Sergey1011010**
**Requires a change from frontend side**

# Description

Fixed a bug with ignored error key

Fixes #209

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update
- Refactor (refactoring code, removing useless files)

## Testing steps

CI

### CODE REVIEW

**Comment by tisnik** [comment]
Please rebase first

File: openapi.json [hunk 1]
```diff
@@ -384,6 +384,14 @@
               "type": "string"
             },
             "example": "some.python.module"
+          },
+          {
+            "name": "errorKey",
```
Comment (joselsegura) [inline openapi.json:389]: Add a description and example, please

**Comment by Sergey1011010** [comment]
Turning into a draft cuz it's gonna break everything, we need to wait for this https://github.com/RedHatInsights/insights-results-smart-proxy/issues/210 to be fixed

File: openapi.json (hunk 1 above)
Comment (Sergey1011010) [reply openapi.json:389]: done

**Review by quarckster — APPROVED** [review]

**Comment by Sergey1011010** [comment]
I hope the fix is already in master, so closing this one

**Pull request closed without merging**